src/Sarif/Core/SarifLog.cs

```diff
@@ -91,8 +92,18 @@ public static SarifLog Load(Stream source, bool deferred = false)
                 throw new ArgumentException($"File path does not exist: '{filePath}'", nameof(filePath));
             }
 
-            using Stream fileStream = fileSystem.FileOpenRead(filePath);
-            await Post(postUri, fileStream, httpClient);
+            byte[] fileBytes = fileSystem.FileReadAllBytes(filePath);
+            SarifLog sarifLog = Load(new MemoryStream(fileBytes));
+
+            if (!ShouldSendLog(sarifLog))
+            {
+                Console.WriteLine($"Posting of the log file to {postUri} was skipped because the result is empty.");
```

Comment:
>Posting of the log file to {postUri} was skipped because the result is empty. [](http://example.com/codeflow?start=36&length=77)

Change to :

"Post of log file to {postUri} was skipped because the log contains no results."